### PR TITLE
Set decimals on y axis of poverty chart dynamically

### PR DESCRIPTION
**Pull request by yuantaoz**

### CODE REVIEW

**Comment by yuantaoz** [comment]
Examples:
Before
Screen Shot 2022-04-22 at 00 43 22
After
Screen Shot 2022-04-22 at 00 43 51


Before
Screen Shot 2022-04-22 at 00 46 47
After
Screen Shot 2022-04-22 at 00 48 04

**Comment by MaxGhenis** [comment]
Great find with `",~%"`! I didn't know about that d3 formatter. For posterity, here's some info on it from https://github.com/d3/d3-format:

>The `~` option trims insignificant trailing zeros across all format types. This is most commonly used in conjunction with types `r`, `e`, `s` and `%`. For example:

```js
d3.format("s")(1500);  // "1.50000k"
d3.format("~s")(1500); // "1.5k"
```

Could you add a changelog entry?

**Comment by yuantaoz** [comment]
That's exactly the resource! Sure, let me do it.

**Comment by nikhilwoodruff** [comment]
Thanks @yuantaoz - LGTM!

**Review by MaxGhenis — APPROVED** [review]

File: changelog.yaml [hunk 1]
```diff
@@ -612,3 +612,7 @@
     - OpenFisca-US bumped.
     - Federal tax expandable on the US household page.
   date: 2022-04-22 17:56:09
+- bump: minor
```
Comment (MaxGhenis) [inline changelog.yaml:615]:
```suggestion
- bump: patch
```

**Pull request merged**